### DISCUSSION TITLE
Duplicate Bootstrap css

### DISCUSSION THREAD

**Discussion by skiabox** [vercel/next.js#17126] [Help] [answered]
Hello guys.
I am new to Next.js and I am reading this tutorial (https://strapi.io/blog/nextjs-react-hooks-strapi-food-app-1).
So I have a question on it.
The author is injecting bootstrap minified css two times.
One at Layout.js and the second one inside _app.js
Is it some kind of mistake or not?
Thank you very much!

**Comment by jamesmosier** [comment, marked as answer]
You're correct, it should only be in there once. Although it would be possible to use the `key` property to ensure no duplicate tags are rendered, [per the docs](https://nextjs.org/docs/api-reference/next/head). But in this case the `<link />` tag will import bootstrap css twice.

**Reply by skiabox** [reply]
Thank you very much sir for confirming my suspicions on this!
